prime-router/src/main/kotlin/azure/ApiSearch.kt

```diff
@@ -0,0 +1,196 @@
+package gov.cdc.prime.router.azure
+
+import com.microsoft.azure.functions.HttpRequestMessage
+import gov.cdc.prime.router.common.JacksonMapperUtilities
+import org.jooq.Condition
+import org.jooq.DSLContext
+import org.jooq.Field
+import org.jooq.Record
+import org.jooq.SelectJoinStep
+import org.jooq.SortField
+import org.jooq.TableField
+
+data class ApiSearchResult<T>(val totalCount: Int, val filteredCount: Int, val results: List<T>)
+enum class SortDirection {
+    ASC,
```

Comment:
I tried putting something else besides these values and while there was a stack trace in the logs, there was no actual error returned to the user, it just didn't return anything. I  think ideally we would have validation around things like this, but maybe @arnejduranovic can weigh in

Reply:
I'll add a handler to the new endpoint, but I think each endpoint will need to be responsible for converting the exception into a `400`, but I'll put some thought into it.

Reply:
Ok, I've updated the endpoint to catch parsing exceptions and return a `400`